src/gametest/java/ca/teamdman/sfm/gametest/SFMCorrectnessGameTests.java

```diff
@@ -1324,6 +1330,9 @@ public static void falling_anvil_disenchant(GameTestHelper helper) {
     public static void falling_anvil_xp_shard(GameTestHelper helper) {
         helper.setBlock(new BlockPos(1, 2, 1), Blocks.OBSIDIAN);
         var pos = helper.absoluteVec(new Vec3(1.5, 3.5, 1.5));
+        var restore = SFMConfig.COMMON.levelsToShards.get();
+        SFMConfig.COMMON.levelsToShards.set(SFMConfig.Common.LevelsToShards.JustOne);
+        helper.getLevel().getEntitiesOfClass(ItemEntity.class, new AABB(helper.absolutePos(new BlockPos(1, 4, 1))).inflate(3)).forEach(e -> e.discard());
```

Comment:
add a comment saying this is to clean up previous runs (assuming that is why this is here)